components/datepicker/utils.js

```diff
@@ -0,0 +1,12 @@
+export const convertModifiersToClassnames = (modifiers, theme) => {
+  if (!modifiers) {
+    return;
+  }
```

Comment:
PP: It might be more predictable to initialise modifiers as an empty object if it's passed as `undefined`. That way this function will always return an object making it more consistent instead of sometimes an object, and sometimes `undefined` in the case where `modifiers` is `undefined`.